src/compiler/VendorCompiler.ts

```diff
@@ -0,0 +1,379 @@
+import {CompilationResult, ICompiler} from "./interfaces";
+import * as fs from 'fs';
+import * as path from "path";
+import {NPMResolver} from "../model/NPMResolver";
+import {cgreen, formatDuration} from "../utils";
+import {Configuration} from "webpack";
+import * as webpack from "webpack";
+import spawn = require("cross-spawn");
+import * as crypto from "crypto";
+import {CompressCssCompiler} from "./CompressCssCompiler";
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+
+export class VendorCompiler implements ICompiler {
+    public static readonly DEST_JS_DIR = 'generated_js';
```

Comment:
`generated_js` identifier already exists in CplaceTypecriptCompiler
`    public static readonly DEST_DIR = 'generated_js';`
🤔  It might make sense to refactor instead of defining twice